oauth2client/contrib/appengine.py

```diff
@@ -477,15 +469,15 @@ def _parse_state_value(state, user):
         user: google.appengine.api.users.User, The current user.
 
     Raises:
-        InvalidXsrfTokenError: if the XSRF token is invalid.
+        ValueError: if the XSRF token is invalid.
```

Comment:
Sorry, I saw that part about using a custom exception (which I'm leaning towards), but overlooked the privatized part.  I'm thinking to keep it at module-scope, but private (`appengine._InvalidXsrfTokenError`)?  The method that calls it is module-scope and private.